modules/database/models/version.py

```
@@ -0,0 +1,9 @@
+from modules.database.base.imports import *
```

Ruff (0.8.0)

`from modules.database.base.imports import *` used; unable to detect undefined names

(F403)